src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -125,4 +131,22 @@ private function findRouteNameContaining(string $name, RouteCollection $routes):
 
         return $foundRoutesNames;
     }
+
+    private function sortRoutes(RouteCollection $routes, string $propertyName): RouteCollection
+    {
+        $sortedRoutes = $routes->all();
+        if ('name' === $propertyName) {
+            ksort($sortedRoutes);
+        } elseif ('path' === $propertyName) {
+            uasort($sortedRoutes, function ($a, $b) { return $a->getPath() <=> $b->getPath(); });
```

Comment:
please add the type for $a and $b

Reply:
if routes have the same path but different prio, this can result is routes getting mixed up. sorting will only be stable in php 8: https://wiki.php.net/rfc/stable_sorting
So we would need to implement the stable part ourselves prio to php 8.

Reply:
Ok, I'll refactor the test to verify the stability